### DISCUSSION TITLE
如何知道自己的電腦應下載32bit的執行檔，還是64bit的執行檔(1/2)

### DISCUSSION THREAD

**Discussion by enoch-lightning** [1101-GS4719-C/Python-Community-GS4719-C#138] [Ideas]
我們在安裝軟體時，都會需要下載執行檔。若是windows作業系統，執行檔的最後面，也就是副檔名是.exe。電腦的作業系統有分成32bit與64bit。64bit的作業系統，其功能比32bit的作業系統強大。雖較新的電腦都會是64bit的作業系統，但還是確認一下較保險。查看電腦是32bit作業系統，還是64bit作業系統的方法共有兩個，我現在分享第一個。
# 方法一
windows key->以滑鼠左鍵點擊「設定」一下->以滑鼠左鍵點擊「系統」一下->以滑鼠左鍵點選「關於」一下
## 第一步
<img width="959" alt="第一步" src="https://user-images.githubusercontent.com/86505454/139365078-a68a3c9e-7d93-4b5c-9d2a-5e23b44b4749.png">

## 第二步
<img width="960" alt="第二步" src="https://user-images.githubusercontent.com/86505454/139365114-4ef6783d-dfdc-4616-8c90-4df8a4c90d3f.png">

## 第三步
<img width="959" alt="第三步" src="https://user-images.githubusercontent.com/86505454/139365377-a5ac8a9a-ab0d-4446-a289-b073792f6ea8.png">

## 第四步
<img width="960" alt="第四步" src="https://user-images.githubusercontent.com/86505454/139365495-5450011c-78c9-4b9b-9316-f18d6ca80ea9.png">

## 得知電腦作業系統類型
<img width="960" alt="得知電腦的作業系統" src="https://user-images.githubusercontent.com/86505454/139365561-0314fbad-7144-4250-8e09-ddffc262d21b.png">
「64位元」就是「x64」就是「64bit」，因此此電腦為64bit的作業系統。

**Comment by Jie2222** [comment]
謝謝同學細心講解，記得剛開始自己安裝程式的時候也是上網查要怎麼看(哈哈

**Comment by HSU-YU-JEN** [comment]
超詳細的解說，我之前都是一律載64bit的執行檔，也沒有想過原來這跟電腦配置有關

**Reply by enoch-lightning** [reply]
這代表我寫這一篇文章有幫到你。我會繼續努力寫文章。:sheep:

**Comment by Isongzhe** [comment]
感謝分享，我記得現在的電腦好像都是64位元的。32很不常見了
